tardis/montecarlo/montecarlo_numba/formal_integral_cuda.py

```diff
@@ -475,23 +475,23 @@ def trapezoid_integration_cuda(arr, dx):
 
 
 @cuda.jit(device=True)
-def intensity_black_body_cuda(nu, T):
+def intensity_black_body_cuda(nu, temperature):
     """
     Get the black body intensity at frequency nu
-    and temperature T
+    and temperature
 
     Parameters
```

Comment:
The first line of the docstring is a "short summary" and should be in one line. https://numpydoc.readthedocs.io/en/latest/format.html#parameters